go.mod

```diff
@@ -4,11 +4,17 @@ require (
 	github.com/DATA-DOG/go-sqlmock v1.3.3
 	github.com/coreos/go-semver v0.3.0
 	github.com/go-sql-driver/mysql v1.4.1
-	github.com/pingcap/check v0.0.0-20191107115940-caf2b9e6ccf4
-	github.com/pingcap/errors v0.11.0
-	github.com/pingcap/log v0.0.0-20191012051959-b742a5d432e9
+	github.com/golang/snappy v0.0.1 // indirect
+	github.com/juju/errors v0.0.0-20190930114154-d42613fe1ab9 // indirect
+	github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712
+	github.com/pingcap/errors v0.11.5-0.20190809092503-95897b64e011
+	github.com/pingcap/goleveldb v0.0.0-20191226122134-f82aafb29989
```

Comment:
em, I am so surprised that we introduce goleveldb and tidb

Reply:
the tidb dependency is needed because `tidb-tools` depends on it

the goleveldb dependency is indeed unnecessary, and can be `go mod tidy`ed out.

```diff
diff --git a/go.mod b/go.mod
index d3e50bc..71ba166 100644
--- a/go.mod
+++ b/go.mod
@@ -5,10 +5,8 @@ require (
        github.com/coreos/go-semver v0.3.0
        github.com/go-sql-driver/mysql v1.4.1
        github.com/golang/snappy v0.0.1 // indirect
-       github.com/juju/errors v0.0.0-20190930114154-d42613fe1ab9 // indirect
        github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712
        github.com/pingcap/errors v0.11.5-0.20190809092503-95897b64e011
-       github.com/pingcap/goleveldb v0.0.0-20191226122134-f82aafb29989
        github.com/pingcap/log v0.0.0-20200117041106-d28c14d3b1cd
        github.com/pingcap/tidb v1.1.0-beta.0.20200305075918-dcb8b3d5ddf0
        github.com/pingcap/tidb-tools v3.0.11+incompatible
```

Reply:
IMO, I hate this, why dumpling needs to depend on TiDB. But if we have no way to improve it, I can accept now. 

@GregoryIan

Reply:
because it depend on tidb-tools, and tidb-tools need tidb, I also don't like it